components/soldiers/SoldierGallerySection.tsx

```diff
@@ -21,29 +27,31 @@ export function SoldiersGallerySection({ soldier }: { soldier: Soldier }) {
         ></img>
       </div>
       <div className="col-span-12 text-center sm:col-span-11 sm:text-left">
-        <p className="text-2xl">{soldier.name}</p>
+        <p className="text-2xl">{name}</p>
         <p>
           HP: {soldier.baseHp} / ATK: {soldier.baseAtk} / DEF: {soldier.baseDef}{" "}
           / MDEF: {soldier.baseMdef}
         </p>
         <p>
           Move: {soldier.move} / Range: {soldier.range}
         </p>
-        <p className="whitespace-pre-line">{soldier.effect}</p>
+        <p className="whitespace-pre-line">{effect}</p>
       </div>
 
       <div className="flex justify-center col-span-12 border-t-2 border-black border-solid mt-6">
         {soldier.trainingSkill && (
           <button onClick={handleShowMoreButton} className="">
-            <img
-              src="ui/arrow_down.svg"
-              alt="arrow image"
-              className={
-                showMore
-                  ? "transform -rotate-180 bg-white -mt-6"
-                  : "bg-white -mt-6"
-              }
-            />
+            {
+              <ArrowDropDownCircleOutlined
+                className="text-4xl -mt-6 bg-white"
+                //tailwind classes not working here. mui overwrite it
```

Comment:
MUI has a fontSize prop, did that not work?

https://mui.com/api/icon/

Reply:
fixed